frontend/packages/data-portal/app/i18n.ts

```diff
@@ -78,10 +84,17 @@ export const i18n = {
   submitFeedback: 'Submit Feedback',
   terms: 'Terms',
   termsOfUse: 'Terms of Use',
+  tiltQuality: 'Tilt Quality',
+  tiltRange: 'Tilt Range',
+  tiltRangeValue: (min: number, max: number) => `${min}° to ${max}°`,
```

Comment:
I have another version of this but more generalized in my PR, what are your thoughts on it?